pkgs/applications/misc/tasknc/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchurl, taskwarrior, perl, ncurses }:
+
+stdenv.mkDerivation rec {
+  version = "0.8";
+  name = "tasknc-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/mjheagle8/tasknc/archive/v${version}.tar.gz";
+    sha256 = "0max5schga9hmf3vfqk2ic91dr6raxglyyjcqchzla280kxn5c28";
+  };
+
+  #
+  # I know this is ugly, but the Makefile does serious bullshit in this package,
+  # so we have to:
```

Comment:
I guess I also have to apologize for my language. I was a bit... frustrated while writing the package def. I'm sorry.